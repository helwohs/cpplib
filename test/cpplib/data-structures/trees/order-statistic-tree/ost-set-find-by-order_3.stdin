10
10 1 3 10 6 10 2 2 2 10
2 3 7 6 5 4 1 2 2 1
