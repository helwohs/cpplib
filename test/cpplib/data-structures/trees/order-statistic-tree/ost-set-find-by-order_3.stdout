1 10 2 2 2 10 10 6 10 3 
